sdk/identity/identity-web/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "@azure/identity-web",
+  "version": "1.0.0-beta.1",
+  "sdk-type": "client",
+  "description": "Use the Azure Account extension for Visual Studio Code to authenticate with Azure Identity",
+  "main": "dist/index.js",
+  "module": "dist-esm/identity-web/src/index.js",
+  "types": "./types/identity-web.d.ts",
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:samples": "echo skipped",
+    "build:test": "tsc -p . && dev-tool run bundle",
+    "build": "npm run clean && npm run extract-api && tsc -p . && dev-tool run bundle",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* types *.tgz *.log",
+    "execute:samples": "echo skipped",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "echo skipped",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "pack": "npm pack 2>&1",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "echo skipped",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/identity/src",
+    "dist-esm/identity-web/src",
+    "types/identity-web.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "repository": "github:Azure/azure-sdk-for-js",
+  "keywords": [
+    "azure",
+    "cloud",
+    "active directory",
+    "authentication",
+    "credential",
+    "certificate",
+    "web",
+    "account"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity-web/README.md",
+  "sideEffects": false,
+  "dependencies": {
+    "@azure/identity": "2.1.0-beta.2",
```

Comment:
This version of Identity doesn't contain the fix for adding the "public app" to MsalAuthorizationCode's doGetToken and getAuthCodeUrl methods. Adding the public app fixes the "no response" issue.